README-DEV.rst

```diff
@@ -272,6 +266,50 @@ to keep those databases around for further investigation::
     MANGO_TESTS_KEEP_DBS=please \
       make mango-test MANGO_TEST_OPTS='03-operator-test'
 
+Running Clouseau
+~~~~~~~~~~~~~~~~
+
+When configured with the ``./configure`` script, the ``dev/run``
+script is capable of launching Clouseau instances alongside the
+CouchDB nodes and hooking them up.  This is what the ``mango-test``
+and ``elixir-search`` targets also use to run their respective test
+suites and let Clouseau automatically managed for them.
+
+Although the ``./configure`` and the ``dev/run`` scripts try to take
```

Comment:
same here, do we want to use `./dev/run`?

Reply:
Right.